tests/ai_safety/guardrails/conftest.py

```
@@ -32,11 +32,12 @@
     TEST_TLS_PRIVATE_KEY,
 )
 from tests.ai_safety.image_constants import AiSafetyImages
+from tests.fixtures.inference import get_or_create_isvc  # noqa: NIT001
```

Ruff (0.16.1)

[warning] Invalid rule code in `# noqa`: NIT001

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)